fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/flight_model.cfg

```diff
@@ -77,7 +79,7 @@ static_pitch = -0.23 ; degrees, pitch when at rest on the ground (+=Up, -=Dn)
 static_cg_height = 8.89 ; feet, altitude of CG when at rest on the ground
 gear_system_type = 3 ; gear system type (betweeen 0 and 4) 0 = electrical, 1 = hydraulic, 2 = pneumatic, 3 = manual, 4 = none, 5 = undefined
 tailwheel_lock = 0 ; Is tailwheel lock available TRUE/FALSE
-max_number_of_points = 23 ; Number of contact points
+max_number_of_points = 11 ; Number of contact points
```

Comment:
I assume this change was not intended? This was changed as part of another PR recently, most likely you worked on an out-of-date flight_model.cfg

Reply:
Was still working on them. Think I got them all now.

Reply:
looks good now, thanks!